dao/src/main/java/com/blog/dao/jdbc/PostDaoImpl.java

```diff
@@ -216,6 +226,25 @@ public boolean addViewToPost(Long postId) {
         return jdbcTemplate.update(addViewSql, parameterSource) == 1;
     }
 
+    public boolean deleteAllTagsInPost(Long postId) {
+        LOGGER.debug("Delete all tags from post id = [{}] in database.", postId);
+        MapSqlParameterSource parameterSource = new MapSqlParameterSource(ID, postId);
+        return jdbcTemplate.update(deleteAllTagsInPostSql, parameterSource) != 0;
+    }
+
+    public boolean addTagsToPost(Long postId, List<Long> tags) {
+        LOGGER.debug("Add tags to post by id = [{}], tags = [{}]", postId, tags);
+        SqlParameterSource[] parameterSource = SqlParameterSourceUtils.createBatch(postId, tags);
```

Comment:
Looks like neither me nor checkstyle accept existence of this variable here - it isn't used